airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.5.4
```

Comment:
`0.6.0`? I was careless on my bump to `0.5.2` but according to semver, we should reset patch to `0` when we bump a minor version. And presumably you meant to move minor to 6.

Reply:
yes, fixed!